src/components/CardFooter/CardFooter.jsx

```diff
@@ -24,7 +27,7 @@ const CardFooter = ({ title, subtitle, label }) => {
           onClick={() => openModal(modalType.edit)}
           text='Edit'
           theme={buttonThemes.confirm}
-          propStyles={{marginLeft: '20px'}} />
+          propStyles={{ marginLeft: '20px' }} />
```

Comment:
you do understand that this prop (even though it's always the same) will trigger rerender in child component after it was rerendered itself, don't you? I do not force you to change it, but just to acknowledge the fact of it